streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -280,7 +299,8 @@ public void recordThroughput(final Sensor sensor, final long value) {
     }
 
 
-    private Map<String, String> constructTags(final String scopeName, final String entityName, final String... tags) {
+    private Map<String, String> constructTags(final String scopeName, final String entityName,
+                                              final String... tags) {
```

Comment:
nit: one line per parameter (although this looks maybe short enough for one line?)